website/docs/language/resources/syntax.mdx

```diff
@@ -87,41 +84,27 @@ The values for resource arguments can make full use of
 [expressions](/terraform/language/expressions) and other dynamic Terraform
 language features.
 
-There are also some _meta-arguments_ that are defined by Terraform itself
-and apply across all resource types. (See [Meta-Arguments](#meta-arguments) below.)
+[Meta-Arguments](#meta-arguments) are defined by Terraform itself
+and apply across all resource types. 
 
 ### Documentation for Resource Types
 
 Every Terraform provider has its own documentation, describing its resource
 types and their arguments.
 
-Most publicly available providers are distributed on the
-[Terraform Registry](https://registry.terraform.io/browse/providers), which also
-hosts their documentation. When viewing a provider's page on the Terraform
-Registry, you can click the "Documentation" link in the header to browse its
-documentation. Provider documentation on the registry is versioned, and you can
-use the dropdown version menu in the header to switch which version's
-documentation you are viewing.
+Some provider documentation is still part of Terraform's core documentation 
+but the [Terraform Registry](https://registry.terraform.io/browse/providers) 
+is now the main home for all publicly available provider docs.
 
-To browse the publicly available providers and their documentation, see
-[the providers section of the Terraform Registry](https://registry.terraform.io/browse/providers).
-
--> **Note:** Provider documentation previously existed as part of Terraform's core documentation. Although some provider documentation
-might still be hosted here, the Terraform Registry is now the main home for all
-public provider docs.
-
-## Resource Behavior
-
-For more information about how Terraform manages resources when applying a
-configuration, see
-[Resource Behavior](/terraform/language/resources/behavior).
+When viewing a provider's page on the Terraform
+Registry, you can click the "Documentation" link in the header to browse its
+documentation, which is versioned. Use the dropdown version menu in the header 
+to switch the version.
```

Comment:
```suggestion
When viewing a provider's page on the Terraform
Registry, you can click the **Documentation** link in the header to browse its
documentation. The documentation is versioned. To choose a different version of the provider documentation, click on the version in the provider breadcrumbs to choose a version from the drop-down menu.
```
Some of these are existing style guide violations, so thanks for opening this PR and giving us the mandate to fix them.